Source/Urho3D/Graphics/Skeleton.h

```diff
@@ -110,14 +110,20 @@ class URHO3D_API Skeleton
 
     /// Return root bone.
     Bone* GetRootBone();
+    /// Return index of the bone by name hash. Return number of bones if not found.

```

Comment:
It is customary for Urho to return M_MAX_UNSIGNED when something is not found.

Reply:
Ok I'll do it.
BTW, `Vector::IndexOf` behaves like iterator, so it returns numElements. Is't probably an exception

Reply:
I don’t think so. Also it’s easier for caller to check against the constant rather than making another call to get the actual size in order to know if the return value is out of bound.